GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -395,36 +556,66 @@ private void SetRowCount(int count)
             }
         }
 
+        /// <summary>
+        /// Reset the calculated indices.
+        /// </summary>
+        private void ResetGraphIndices()
+        {
+            _toBeSelectedGraphIndexesCache = new(() => CalculateGraphIndices());
+        }
+
+        /// <summary>
+        /// Get the revision graph row indexes for the ToBeSelectedObjectIds.
+        /// (In filtering situations, all may no longer be in the grid).
+        /// </summary>
+        private IList<int> CalculateGraphIndices()
+        {
+            Debug.Assert(_loadedToBeSelectedRevisionsCount == ToBeSelectedObjectIds.Count,
+                "GetGraphIndexes() was called before all expected revisions were loaded.");
```

Comment:
Is this outdated? Should this be
```suggestion
                $"{nameof(CalculateGraphIndices)}() was called before all expected revisions were loaded.");
```

Reply:
changing